### DISCUSSION TITLE
Additional playlist formats

### DISCUSSION THREAD

**Discussion by jagrosh** [jagrosh/MusicBot#604] [Ideas]
Along the lines #585 and #596, there are lots of playlist formats out there that JMusicBot could (in theory) support. If you have a playlist format that you would like to see supported, please comment on this thread with:
1. The name of the format (and its origin/what it's used for)
2. An example playlist file in this format
3. (Optionally) a link to documentation of the format

**Comment by lakshaycodes** [comment]
m3u. it is used in windows media player and if it is supported it can inderictly support many formats like zpl and wpl

**Reply by jagrosh** [reply]
Can you provide an example file or documentaion
